src/store/lokapi.ts

```diff
@@ -93,14 +93,32 @@ export var moduleLokAPI = {
       let accounts: any;
       try {
         accounts = await lokApiService.getAccounts();
-        let balance = await accounts[0].getBalance();
-        let symbol = await accounts[0].getSymbol();
-        state.bal = balance;
-        state.curr = symbol;
+        if (accounts.length > 0) {
+          // Compute a global moneyAccount balance
+          Promise.allSettled(accounts.map((a:any) => {
+            return a.getBalance()
+          }))
+          .then((balances:any) => {
+            state.bal = balances.reduce((s:number, b:any) => {
+              return s + parseFloat(b.value)
+            }, 0);
+          })
+          // See if all accounts have same currency ?
+          Promise.allSettled(accounts.map((a:any) => {
+            return a.getSymbol()
+          }))
+          state.curr = await accounts.reduce((c:string, a:any) => {
+            let s = a.value
+            if (s !== c) {
+              return 'mixed currencies'
```

Comment:
Note that you are introducing here an english string that will be displayed to the currently french interface.

Reply:
Of entendu... C'est le problème when you want to go too fast and do not réellement attention to which language you are en train d'écrire en. ;-)
Will try and slow down and pay more attention.

Reply:
As you probably noticed, I removed the total that was displayed thanks as the Global component was not wanted anymore. Sums are made, but only for user accounts (in one currency).